smart_proxy_pulp.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency('webmock', '~> 1')
   gem.add_development_dependency('rack-test', '~> 0')
   gem.add_development_dependency('rake', '~> 10')
+  gem.add_dependency('sys-filesystem', '~> 1.1')
```

Comment:
Uh are you sure? :-) You need to create packages for Red Hat and Debian... I was not insisting on this at all.